test/parallel/test-buffer-safe-unsafe.js

```diff
@@ -7,8 +7,18 @@ const safe = Buffer.alloc(10);
 
 function isZeroFilled(buf) {
   for (let n = 0; n < buf.length; n++)
-    if (buf[n] > 0) return false;
+    if (buf[n] !== 0) return false;
```

Comment:
Yeah, that came up in the original review. It's not a great test. The plan was to revisit to see if the test can be made more robust.